src/environments/environment.ts

```diff
@@ -1,9 +1,28 @@
-// The file contents for the current environment will overwrite these during build.
-// The build system defaults to the dev environment which uses `environment.ts`, but if you do
-// `ng build --env=prod` then `environment.prod.ts` will be used instead.
```

Comment:
according to this, does the Prod pipeline expects environment.prod.ts to be an existing file, which we are now removing?

Reply:
reverted environment.prod.ts. The script set-env.ts will now populate environment.prod.ts on build